packages/react-next/etc/react-next.api.md

```diff
@@ -992,18 +992,24 @@ export class PivotItem extends React.Component<IPivotItemProps, {}> {
     render(): JSX.Element;
 }
 
-// @public (undocumented)
-export enum PivotLinkFormat {
-    links = 0,
-    tabs = 1
+// @public @deprecated (undocumented)
+export const enum PivotLinkFormat {

```

Comment:
hm why did you choose to make this change now? I feel we should avoid this change since there are already enough changes for us to worry about (need codemods/shim)

Reply:
This is updating the `linkFormat` and `linkSize` props to use the current pattern of just being a string with type restrictions instead of an enum.

The goal was for it to be backwards compatible without requiring a codemod. By redefining the enum values to be the same strings, existing code should work without modification. Anywhere that sets e.g. `linkFormat={PivotLinkFormat.tabs}` will now effectively be setting `linkFormat="tabs"` without requiring any code changes. But let me know if I'm wrong about that? Perhaps adding `@deprecated` could cause problems?

I could avoid modifying the API and convert the enum to a string internally, but I figured that since this was backwards compatible it would be nice to improve it while I'm modifying it. Let me know if you still disagree and I'll revert it.

Reply:
The motivating reason is to make this prop be seamlessly compatible with style modifiers (`linkFormat_tabs`), but it's not necessary to modify the API for just that.

Reply:
I think this seems really nice. I think it should be backwards compatible, but is more the ideal setup. Devs can just pass in strings without importing enums.

Reply:
ic now. sorry i mis-read this as a breaking change